package.json

```
@@ -4,9 +4,11 @@
   "main": "src/handler.js",
   "scripts": {
     "test": "jest",
+    "format": "prettier --write .",
+    "lint": "xo .",
+    "check": "xo && jest",
```

These are great! Personally I'd like a command that does everything that I can run as a "final check" before doing a push, how would we feel about making `check` do prettier -> xo -> jest?

```suggestion
    "format": "prettier --write .",
    "lint": "xo .",
    "check": "prettier --write && xo && jest",
```

---

Your suggestion will unfortunately not run. I've done some googling to try and work out how to do this, as I also think it would be good to have in one script, but haven't found out how to do it.

---

[This page](https://gist.github.com/coryhouse/b26f49bead69066844d9) led me to a solution that seems to work:

```suggestion
    "format": "prettier --write .",
    "lint": "xo .",
    "check": "npm run format && npm run lint && npm run test"",
```